VirtoCommerce.InventoryModule.Data/Model/FulfillmentCenterEntity.cs

```diff
@@ -11,7 +10,7 @@ public class FulfillmentCenterEntity : AuditableEntity
         [StringLength(128)]
         public string Name { get; set; }
 
-        public string Description { get; set; }     
+        public string Description { get; set; }
```

Comment:
Seems there are no real changes, could be reverted

Reply:
reverted